kernel/byterun/make_jumptbl.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+sed -n -e '/^  /s/ \([A-Z]\)/ \&\&coq_lbl_\1/gp' -e '/^}/q' ${1} > ${2}
```

Comment:
does this file need to exist? can't you call sed from dune (as we do in Make)?

Reply:
I think it should be the other way around and Make should call this file instead of inlining code, so we share the definition.

I could indeed inline the code tho.

Reply:
This is a good idea. Please call the two .sh scripts from Make, so that there is no chance of getting the two build systems out of sync.